src/main/java/com/meraphorce/controllers/UserController.java

```diff
@@ -16,12 +25,120 @@ public class UserController {
     private UserService userService;
 
     @PostMapping
-    public ResponseEntity<User> createUser(@RequestBody User user){
-        return ResponseEntity.ok(userService.createUser(user));
+    public ResponseEntity<ResponseDTO<?>> createUser(@RequestBody UserDTO user) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	UserDTO newUserData = null;
+    	
+    	newUserData = this.userService.createUser(user);
+    	
+    	if(newUserData != null) {
+    		responseDTO = ResponseDTO.builder()
+    				.object(newUserData)
+    				.status(true)
+    				.build();
+    	} else {
+    		responseDTO = ResponseDTO.builder()
+    				.message("Error to create user.")
+    				.status(false)
+    				.build();
+    	}
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+        return response;
     }
 
     @GetMapping
-    public ResponseEntity<List<User>> getAllUsers(){
-        return ResponseEntity.ok(userService.getAllUsers());
+    public ResponseEntity<ResponseDTO<?>> getAllUsers() {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	List<UserDTO> listUsers = null;
```

Comment:
The same with the create , you can simplify in one statement like this List<UserDTO>  listUsers = this.userService.getAllUsers();